How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md

```diff
@@ -0,0 +1,251 @@
+# How to Build a Flask, ReactJS Application and Deploy to Okteto
+## Introduction 
+
+**[Flask](https://flask.palletsprojects.com/en/2.0.x/)** is a great open source web microframework built with Python. It is straightforward and easy to understand.
+Flask is used to build web applications and also APIs. In this article we will use it to build an API that displays date and time.
+
+ReactJS is an open-source  front-end  JavaScript library used to build user interfaces.
+
+**[Okteto](https://okteto.com/)** is an open-source tool that runs locally to synchronize application code changes to a running pod in a local or remote Kubernetes cluster. With Okteto we are able to see deployment version of our app as we are developing it.
+Okteto has a development environment which we are going to use instead of working locally. Of course we can build this project with our development but we’ll be using Okteto's development environment to build our app which gives us a production-like development environment.
+
+In this article we will build an application that displays the date and time with React and Flask and deployment will be handled with Okteto
```

Comment:
Do you mind building another sample for this draft? A to-do list application or an application with a bit of complexity will be better off than a date and time display application.

Reply:
I stated the project I will be building in the outline but there were no queries.
But no worries, I'll make the appropriate changes.

Reply:
Hi, I'm sorry I might have missed the changes you made in the outline. Thank you.